### ISSUE TITLE
current command isn't read on windows when navigating in the terminal accessible view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#215579]
reported by @jooyoungseo

**Comment by meganrogge** [comment]
@jooyoungseo, I am calling `alert` here on navigating to the command. On macOS, this works beautifully. I haven't tested on Windows, but am not sure why it wouldn't work the same way there. 

One reason it wouldn't work for you is if shell integration isn't enabled or activated properly. I know we've seen that happen for you before.

https://github.com/microsoft/vscode/blob/cfe166664c618eee308177cb4703bee176d21372/src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts#L192

@rperez030, do you have any thoughts / can you repro the issue on windows:
- run some commands in the terminal
- open the accessible view
- use alt/opt+up/down arrow to navigate between commands
- you should hear the commands

**Comment by jooyoungseo** [comment]
@meganrogge It has nothing to do with the shell integration because the cursor actually moves, but the aria alert does not work on Windows.

**Comment by rperez030** [comment]
hi @meganrogge. If I follow the steps, I only hear command failed / command succeeded for each command with the corresponding audio cue, and these are actually communicated as screen reader alerts.

if I do the same on the mac, I hear the command and the corresponding audio cue for success / failure, but not the command succeeded / command failed message. for the sake of troubleshooting, I disabled the signal announcement on Windows for command succeeded / command failed. After doing that I hear the audio cue only, but not announcement. I did a bit of debugging with NVDA, and the screen reader doesn't seem to be receiving any aria live update event.

One easy fix here could be to simply select the text for the command when the cursor moves to previous / next command. That would completely eliminate the need for an aria alert, and would likely work consistently across all platforms.

**Comment by meganrogge** [comment]
Thanks so much for troubleshooting @rperez030. I love that idea, will try it tomorrow.

**Comment by karthiknadig** [comment]
I only hear the command status as "command succeeded" or "command failed" but not the command itself.

**Comment by meganrogge** [comment]
@karthiknadig do you see the command selected? A gif would help

**Comment by karthiknadig** [comment]
@meganrogge it is not working for me because of this: https://github.com/microsoft/vscode/issues/227132

**Comment by meganrogge** [comment]
Makes sense - this feature is built on shell integration, so that'll have to work to test this

**Comment by karthiknadig** [comment]
@Tyriar provided a fix for the shell integration. This is verified with that fix.
